486_Project/Characters_Class.py

```diff
@@ -1,53 +1,77 @@
 class warrior:
```

Comment:
Class Names by convention are capitalized. (i.e. Warrior) 
functions and methods are lowercase